index-pl.html

```diff
@@ -562,25 +562,25 @@ <h2>How To Stop An Epidemic</h2>
       </span>
     </aside>
 
-    <h2>The Human Cost</h2>
+    <h2>Koszt ludzki</h2>
 
     <p>
-      We can take our understanding one step further by expressing the power of masks in more human terms.
-      Masks save lives by reducing the chance of infection which, in turn, shrinks the extent of the epidemic.
+      Możemy poszerzyć nasze rozumienie krok dalej i wyrazić moc masek za pomocą bardziej przystępnych pojęć.
```

Comment:
When you write 'krok dalej' then it is saying 'gdzie' możemy poszerzyć nasze rozumienie. Jeżeli chcemy wyrazić różnicę "odległości", czyli wyrazić to, co autor miał na myśli, sugeruję:
"o krok dalej"